Formula/clickhouse.rb

```diff
@@ -0,0 +1,74 @@
+class Clickhouse < Formula
+  desc "Free analytics DBMS for big data with SQL interface"
+  homepage "https://clickhouse.tech"
+  url "https://github.com/ClickHouse/ClickHouse/releases/download/v21.6.3.14-stable/ClickHouse_sources_with_submodules.tar.gz"
+  version "21.6"
+  sha256 "696bbcdef4796bfdf1becdb78abb5e97c0220acc9863f5a821193dd297ce428b"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/ClickHouse.git"
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on xcode: :build
+
+  def install
+    mkdir_p bin
+    mkdir_p "#{HOMEBREW_PREFIX}/etc/clickhouse-server"
+    mkdir_p "#{var}/log/clickhouse-server"
+    mkdir_p "#{var}/lib/clickhouse-server"
+    mkdir_p "#{var}/run/clickhouse-server"
+
+    inreplace "programs/server/config.xml" do |s|
+      s.gsub! "<!-- <max_open_files>262144</max_open_files> -->", "<max_open_files>262144</max_open_files>"
+    end
+    inreplace "cmake/warnings.cmake" do |s|
+      s.gsub!(/add_warning\(frame-larger-than=(\d*)\)/, "add_warning(frame-larger-than=131072)")
+    end
+
+    args = std_cmake_args
+    args.delete("-DCMAKE_BUILD_TYPE=Release")
+    args << "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+
+    system "cmake", ".", *args
```

Comment:
```suggestion
    system "cmake", ".", *std_cmake_args
```
We don't support building with debug symbols.

Reply:
Haven't met this in Cookbook. I can change it. 
Can you elaborate what is the idea? I don't know how I can help a user to report a bug without symbols and I don't know a way to provide separate package with symbols in brew. In Debian packages we can split artifacts to binary and symbols – do we have alternative there?

Reply:
Not really, though I'm personally open to entertain the possibility of externalising debug information and packaging them separately. (I don't speak for other maintainers on this, though.)

At the moment, however, we don't support building with debug info at all. Indeed, you'll find that most attempts to build with debug info will either break the build, or just not produce any debug info (since `brew` will intercept any `-g` flags passed to the compiler).